src/Uno.UI/UI/Xaml/Controls/Popup/PopupRoot.cs

```diff
@@ -19,6 +19,20 @@ internal partial class PopupRoot : Panel
 	public PopupRoot()
 	{
 		KeyDown += OnKeyDown;
+		Window.Current.Activated += (_, _) => CloseFlyouts();
+		Window.Current.SizeChanged += (_, _) => CloseFlyouts();
+	}
+
+	private void CloseFlyouts()
```

Comment:
Could this be made static?

Reply:
I think this will need to be changed when multiwindow support hits anyway, so I guess it doesn't matter